app/views/layouts/application.html.haml

```diff
@@ -0,0 +1,31 @@
+!!!
+%html
+  %head
+    %meta{:content => "text/html; charset=UTF-8", "http-equiv" => "Content-Type"}/
```

Comment:
The same thing @dannharo told you, use  content: 'text/html' way, instead of the hash rocket way :content => 'navbar-link'